### ISSUE TITLE
(Bug: Kuiper) When uploading new Mars credentials, I have to refresh the page before it correctly shows it's connected

### ISSUE THREAD

**Issue by lamepennies** [juno-fx/report#275]
### Dupe Check

- [X] I have [searched Juno's bugs](https://github.com/juno-fx/report/issues?q=is%3Aopen+is%3Aissue+label%3Abug) and there are no duplicates

### Serivce this pertains to.

Mars

### URL in Hubble.

https://ath.junovfx.com/workstations/

### Browser

Chrome Based

### Describe the bug

When I upload my credentials to Mars, it doesn't show it's connected. I have to refresh the page before it shows up

### To reproduce

Delete old credentials, upload new ones and wait for it to show it's connected. It doesn't so I refresh the page. 

### Regression

No, this bug or issue has existed throughout my experience using Orion

### Does this block you from using Orion daily?

No

**Comment by lamepennies** [comment]
Same issue on SQM. Can't see the files I've uploaded
![Screenshot 2024-10-03 at 10 09 06 PM](https://github.com/user-attachments/assets/26f15b40-0f8c-4bc0-adcb-74f419232397)
![Screenshot 2024-10-03 at 10 09 19 PM](https://github.com/user-attachments/assets/71abd04b-5f80-46ce-92ba-9ed888d570c3)

**Comment by Anthonygen1** [comment]
Do the files show up when you refresh the page as you said in the top comment? Sounds like the code needs to be reworked in hubble to handle this edge case

**Comment by lamepennies** [comment]
Nope, no files. Made a ticket for that here: https://github.com/juno-fx/report/issues/274

**Comment by aldmbmtl** [comment]
This is a Mars issue. This is the pagination issue
